Controllers/DetailedInterventionViewController+Weather.swift

```diff
@@ -65,7 +65,8 @@ extension AddInterventionViewController {
   // MARK: - Actions
 
   func loadWeatherInEditableMode() {
-    let weatherDescriptions = ["BROKEN_CLOUDS", "CLEAR_SKY", "FEW_CLOUDS", "LIGHT_RAIN", "MIST", "SHOWER_RAIN", "SNOW", "THUNDERSTORM"]
+    let weatherDescriptions = ["BROKEN_CLOUDS", "CLEAR_SKY", "FEW_CLOUDS", "LIGHT_RAIN",
```

Comment:
We've got `weatherDescription` defined in 3 different places, can't we extract that to a constant somewhere?